submissions/lalalilulu/hooli-style_popup/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<header>
+    <nav>
+        <ul class="menu">
+            <li>Mail</li>
+            <li>Images</li>
+        </ul>
+        <div class="popup">
+            <img class="popup-icon" src="icons/popup-button.png">
+            <div class="popup-container">
+                    <div class="arrow"></div>
+                    <div class="popup-items">
+                        <div class="popup-item">
```

Comment:
`.popup-item` must be a link. The user should be able to click on the links inside your popup.